### ISSUE TITLE
Node-Schema (start declaration)

### ISSUE THREAD

**Issue by vmaccatrozzo** [labra/shaclex#37]
We would be interested in using this feature. Is it something you'll be working on?

**Comment by labra** [comment]
Yes, it is internally supported but I hadn't added support for it yet externally. 

We will add support for it in the next days.
